src/sentry/static/sentry/app/plugins/components/issueActions.jsx

```diff
@@ -68,6 +80,86 @@ class IssueActions extends PluginComponentBase {
     );
   }
 
+  setDependentFieldState(fieldName, state) {
+    const dependentFieldState = {...this.state.dependentFieldState};
+    dependentFieldState[fieldName] = state;
+    this.setState({dependentFieldState});
+  }
+
+  loadOptionsForDependentField = async field => {
+    const formData = this.getFormData();
+
+    const url =
+      '/issues/' +
+      this.getGroup().id +
+      '/plugins/' +
+      this.props.plugin.slug +
+      '/options/';
+
+    //find the fields that this field is dependent on
+    const dependentFields =
+      fromPairs(field.depends.map(fieldKey => [fieldKey, formData[fieldKey]])) || {};
+    const query = {
+      field: field.name,
+      ...dependentFields,
```

Comment:
idk `dependentFields` data structure, but is it possible it has a `field` property?

Reply:
@billyvg It's based off the name of the field so it's definitely possible if we have a `field` named `field` but that seems unlikely to happen for any plugin and definitely does not for Trello.

Reply:
Still might be good to make sure the line above overrides the spread then

Reply:
@billyvg That would break things as well since if the field is dependent, we need to pass the value. A better solution would be to change `field` to another name. Thoughts?